### ISSUE TITLE
requests.exceptions.ConnectionError

### ISSUE THREAD

**Issue by yahongie2014** [cyrildiagne/ar-cutpaste#11]
How i Can Solve that Problem in server host

![image](https://user-images.githubusercontent.com/5874360/81021504-99ba7500-8e6b-11ea-82f4-7c2a42fe30cf.png)

**Comment by buskerone** [comment]
Hi, I was facing the same issue. Try to add `:8080` to all the requests like this:
`r = requests.get('http://your.ip:8080')`
`res = requests.post('http://your.ip:8080', headers=headers, files=files )`

**Comment by yahongie2014** [comment]
> 
> 
> Hi, I was facing the same issue. Try to add `:8080` to all the requests like this:
> `r = requests.get('http://your.ip:8080')`
> `res = requests.post('http://your.ip:8080', headers=headers, files=files )`

where i can add it im already add it in server.tsx
![image](https://user-images.githubusercontent.com/5874360/81022324-a50ea000-8e6d-11ea-9939-973d64366ff9.png)

**Comment by buskerone** [comment]
You are showing the RN app and it's in the server -> server/src/main.py

**Comment by yahongie2014** [comment]
i already add the port and it  listen on myip:8080

but when i call and route 


requests.exceptions.ConnectionError: HTTPConnectionPool(host='192.168.193.97', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x08460BB0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
Traceback (most recent call last)


![image](https://user-images.githubusercontent.com/5874360/81023180-ec962b80-8e6f-11ea-98b5-10d75d2e9145.png)

![image](https://user-images.githubusercontent.com/5874360/81023161-da1bf200-8e6f-11ea-950c-adec9c47dcc9.png)

**Comment by yahongie2014** [comment]
im pretty sure it cause the ip of remote connection of photoshop !

**Comment by buskerone** [comment]
that host at the bottom shouldn't be changed, put 0.0.0.0 there...

**Comment by yahongie2014** [comment]
then how i loged in !!!

**Comment by yahongie2014** [comment]
!!!!!!!!1
![image](https://user-images.githubusercontent.com/5874360/81023467-cae97400-8e70-11ea-8405-42fcf04c36ca.png)

**Comment by buskerone** [comment]
did you run the app 

PD. can you pl try to avoid -> !!!!  ? this is a github issue and you are a developer..

**Comment by yahongie2014** [comment]
it the same error 
![image](https://user-images.githubusercontent.com/5874360/81023594-2e73a180-8e71-11ea-9893-03dd211d990b.png)

**Comment by buskerone** [comment]
don't put localhost... put the same IP as before. Did you run the app? It's a RN app made with expo. If you don't know what is it then you should read the documentation.

**Comment by yahongie2014** [comment]
yes i already run expo and successes build but i appear black screen with white dots
![image](https://user-images.githubusercontent.com/5874360/81023686-709ce300-8e71-11ea-9135-0393a5240575.png)

**Comment by yahongie2014** [comment]
![](https://i.imgur.com/Dwllgmj.jpg)

**Comment by yahongie2014** [comment]
what i trying to say  backend wont connect correctly cause photoshop ip or thing related python and im sorry if i annoying you im not python developer

**Comment by yahongie2014** [comment]
and api for cut image wont run the same error if you have any idea for config remote photoshop
its will be awesome and thank u gain 

![image](https://user-images.githubusercontent.com/5874360/81024223-13099600-8e73-11ea-8d42-cbaeca76e397.png)

**Comment by jardicc** [comment]
This project uses Connection SDK to connect with Photoshop and it has static port 49494 and password must match on both sides. https://github.com/adobe-photoshop/generator-core/blob/24a2c4e38aef79adb42869f200778ab52de4ef36/lib/photoshop.js#L41 And it is not even HTTP or Rest API. So you are not trying to connect to Photoshop but something else.
